### ISSUE TITLE
Custom ordering for variations not work

### ISSUE THREAD

**Issue by miniMAC** [woocommerce/woocommerce#18974]
WC version: 3.3.1
WP version: 4.9.4

The custom order for variations not work on select2. Variations order still remain ordered by name.

**Comment by miniMAC** [comment]
@mikejolley yes sorry. The problem is that the attributes "values" can't be reordered with custom order.

**Comment by mikejolley** [comment]
It's done via products > attributes globally.
